scripts/build/build_macos.py

```
@@ -173,6 +192,25 @@ def verify_binary() -> bool:
         print("Help check timed out", file=sys.stderr)
         return False
 
+    # Check doctor loads manifest.json (required for tool registry)
+    try:
+        result = subprocess.run(
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)